drivers/serial/uart_npcx.c

```diff
@@ -285,6 +294,16 @@ static int uart_npcx_err_check(const struct device *dev)
 	return err;
 }
 
+#ifdef CONFIG_PM
+static void uart_npcx_rx_wk_isr(const struct device *dev, struct npcx_wui *wui)
```

Comment:
Replace #ifdef CONFIG_PM with __unused static void uart_npcx_rx_wk_isr().   Then you can also use IS_ENABLED() at line 369.

Reply:
Oh, yes. Thanks for reminding me. I will submit the PR with this change later.

Reply:
Done.